### DISCUSSION TITLE
networking/clb-to-pod-directly

### DISCUSSION THREAD

**Discussion by giscus** [imroc/tke-guide#5] [General]
https://imroc.cc/tke/networking/clb-to-pod-directly

**Comment by huangjc7** [comment]
作者你好，我使用nginx-ingressController+hostNetwork模式， 然后通过直接访问节点的ip 这样子理论上来说是直通pod了吗。但是我在conntrack的表里面依旧看到了记录。我有些迷茫 希望可以解惑 感谢🙏

**Reply by imroc** [reply]
这种没经过lb，且使用 hostNetwork，自然相当于直通pod了

**Reply by huangjc7** [reply]
很抱歉我可能昨天的描述太模糊了，我重新描述一下。我将nginx-ingressController的pod设置为hostNetwork，然后编写一段ingress的规则，让nginx-ingressController负载到一个测试应用的pod；
网络流量为 nodeIP -> nginx-ingressController-> podIP 在nodeIP -> nginx-ingressController之间是使用了hostNetwork模式。这样子一个流向

**Reply by imroc** [reply]
直通本身就是针对南北向流量，即外部流量到nginx-ingress-controller这一跳，这里是hostNetwork，nignx-ingress-controller的ip就是nodeIP，那么外部流量到nginx-ingress-controller就是 client-->nodeIP(nginx-ingress-controller)，中间不经过 nodeport/ipvs，也就是直通了

**Reply by huangjc7** [reply]
嗯嗯好的，非常感谢解惑，祝假期愉快
